pkgs/development/python-modules/monitorcontrol/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, poetry-core
+, pyudev
+, pytestCheckHook
+, voluptuous
+}:
+
+buildPythonPackage rec {
```

Comment:
Is this a library? If it is an application and not intended to be used as a library, please use `buildPythonApplication` and move it to pkgs/by-name or somewhere.

Reply:
It's primarily a library with a CLI for testing purposes.

Reply:
I see. Thanks for clarifying.